docs/_source/guides/query_datasets.md

```diff
@@ -86,6 +86,10 @@ Metadata are indexed as keywords.
 This means you cannot search for single words in them, and capitalization and punctuations are taken into account.
 You can, however, use wild cards.
 
+If your intention is to only store metadata with records and not use it for searches, you can achieve this by defining
+the metadata field with a leading underscore. For instance, if you use `metadata._my_hidden_field`, the field will be
+accessible at the record level, but it won't be used in searches.
+
```

Comment:
Maybe it's better if we add a sub-section? (Not sure, please ignore it if it's too much)

```suggestion
### Non-searchable metadata fields

If your intention is to only store metadata with records and not use it for searches, you can achieve this by defining
the metadata field with a leading underscore. For instance, if you use `metadata._my_hidden_field`, the field will be
accessible at the record level, but it won't be used in searches.

```

Reply:
Yes, we can define a subsection. Also, it will be simpler to share it by direct link